test/test_citusdb.yml

```
@@ -0,0 +1,256 @@
+      #with_nested: "{{ query_result.stdout }}"
+ 
+#   - name: Assert results are correct
+#     assert:
+#       that: 
+#         - "{{ create_table_users_sql_results['status'] }} s defined"
+    - name: create table events
+      postgresql_exec:
+        login_user: "{{ pp_superadmin }}"
+        login_password: "{{ pp_superpassword }}"
+        login_host: citus_coordinator
+        port: "{{ pp_serverport }}"
+        db: "postgres"
+        script: "create table IF NOT EXISTS github_events(event_id bigint, event_type text, event_public boolean, 
+          repo_id bigint, payload jsonb,repo jsonb,user_id bigint,org jsonb, created_at timestamp);"
+      register:  create_table_events_sql_results
+ 
+
+    - name: print sql results
+      debug:
+        msg: "{{ create_table_events_sql_results['status'] }}"
+
+    - name: create event index 
+      postgresql_exec:
+        login_user: "{{ pp_superadmin }}"
+        login_password: "{{ pp_superpassword }}"
+        login_host: citus_coordinator
+        port: "{{ pp_serverport }}"
+        db: "postgres"
+        script: "CREATE INDEX IF NOT EXISTS event_type_index ON github_events(event_type);"
+        
+    - name: create payload index 
+      postgresql_exec:
+        login_user: "{{ pp_superadmin }}"
+        login_password: "{{ pp_superpassword }}"
+        login_host: citus_coordinator
+        port: "{{ pp_serverport }}"
+        db: "postgres"
+        script: "CREATE INDEX IF NOT EXISTS payload_index ON github_events USING GIN (payload jsonb_path_ops);"
+        
+    - name: create distributed table users
+      postgresql_exec:
+        login_user: "{{ pp_superadmin }}"
+        login_password: "{{ pp_superpassword }}"
+        login_host: citus_coordinator
+        port: "{{ pp_serverport }}"
+        db: "postgres"
+        script: "SELECT create_distributed_table('github_users', 'user_id');"
+      register:  create_dist_table_users_sql_results
+      ignore_errors: true
```

I think we don't want to ignore errors here, right?  Same with the next two.